components/brandprofile/AnalysisShowcase.tsx

```
@@ -0,0 +1,930 @@
+    <div
+      className={`bg-white dark:bg-secondary-800 rounded-xl p-6 shadow-sm border border-secondary-200 dark:border-secondary-700 ${className}`}
+    >
+      <h4 className="text-lg font-semibold text-secondary-900 dark:text-secondary-100 mb-4">
+        {title}
+      </h4>
+      <div className="grid grid-cols-2 gap-4">
+        {metrics.map((metric, index) => (
+          <div key={metric.label} className="text-center">
+            <div
+              className="w-full h-2 rounded-full mb-2"
+              style={{ backgroundColor: metric.color + '20' }}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)